CHANGELOG.md

```diff
@@ -269,6 +272,7 @@ you can find their release notes
 [1358]: https://github.com/enso-org/ide/pull/1358
 [1377]: https://github.com/enso-org/ide/pull/1377
 [1411]: https://github.com/enso-org/ide/pull/1411
+[1366]: https://github.com/enso-org/ide/pull/1366
```

Comment:
I would put that in a new list, at the end of "Next Release" section.

Reply:
Done.